### PR TITLE
Translate 'Powered by' text

### PR DESCRIPTION
**Pull request by JuraSN**
Footer:

screen

### CODE REVIEW

**Comment by ppoffice** [comment]
@JuraSN could you please resolve the conflict? Thanks.

**Comment by JuraSN** [comment]
Done.

File: languages/ru.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ nav:
   search: 'Поиск'
   toc: 'Оглавление'
 article:
+<<<<<<< HEAD
```
Comment (ppoffice) [inline languages/ru.yml:14]: This shouldn't be here.

Comment (JuraSN) [reply languages/ru.yml]: Corrected.

**Pull request merged**